lib/commands/bin.js

```diff
@@ -1,6 +1,10 @@
 const log = require('../utils/log-shim.js')
-const envPath = require('../utils/path.js')
 const BaseCommand = require('../base-command.js')
+// TODO this may not be needed at all. Ignoring because our tests normalize
+// process.env.path already
+/* istanbul ignore next */
+const path = process.env.path || process.env.Path || process.env.PATH
```

Comment:
TIL (ref: https://nodejs.org/api/process.html#processenv)

> On Windows operating systems, environment variables are case-insensitive.

```
import { env } from 'process';

env.TEST = 1;
console.log(env.test);
// => 1
```

we do this `path/Path/PATH` thing in several places i think, and it looks like maybe they can all go fly a kite